### PR TITLE
Support Ruby 2.7

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: lib/journald/classes/logger.rb [hunk 1]
```diff
@@ -58,7 +58,7 @@ def tag(**tags)
         yield self
       end
     ensure
-      tag(values) if values.any? # restore old values if block given
+      tag(**values) if values.any? # restore old values if block given
```
Comment (lzap) [inline lib/journald/classes/logger.rb:61]: Is this something which fails with ruby 2.7.x ? Can you provide some context?

Comment (mmoll) [reply lib/journald/classes/logger.rb:61]: This is giving a warning:
```
lib/journald/classes/logger.rb:61: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
lib/journald/classes/logger.rb:46: warning: The called method `tag' is defined here
```

Comment (lzap) [reply lib/journald/classes/logger.rb:61]: Thank you, looks like a new release needs to be done.

**Comment by lzap** [comment]
Successfully registered gem: journald-logger (3.0.1)

**Comment by lzap** [comment]
Thank you!

**Pull request merged**